android/app/src/main/java/com/dismoi/scout/Floating/Layout/Message.java

```diff
@@ -0,0 +1,92 @@
+package com.dismoi.scout.Floating.Layout;
+
+import android.content.Context;
+import android.graphics.Point;
+import android.util.AttributeSet;
+import android.util.DisplayMetrics;
+import android.view.Display;
+import android.view.WindowManager;
+
+public class Message extends Base {
```

Comment:
Could `Message` and `Bubble` share at least some of there common properties/behaviour?
Ex: `Positionnable` with `initialX`  `initialY` (or maybe  `Movable` is better or `Draggable` idk)

Reply:
The Base parent class shares the same properties. But I saw I had dead code in the Message layout so i have removed some code